blocks/blogs/blogs.js

```diff
@@ -47,34 +49,88 @@ async function createBlogCard(blogData) {
   articleInfo.appendChild(articleInfoTime);
 
   const articleInfoPoweredBy = document.createElement('abbr');
-  articleInfoPoweredBy.textContent = (await fetchPlaceholders()).icicisecurities;
+  // articleInfoPoweredBy.textContent = (await fetchPlaceholders()).icicisecurities;
+  fetchPlaceholders().then((placeholders) => {
+    articleInfoPoweredBy.textContent = placeholders.icicisecurities;
+  });
   articleInfo.appendChild(articleInfoPoweredBy);
   articleContent.appendChild(articleInfo);
 
   arcticleDiv.appendChild(articleContent);
   return arcticleDiv;
 }
+function formatDateString(dateString) {
```

Comment:
Is the page using similar date format across blocks? If yes, this should likely be a block util method.

Reply:
@sdmcraft API response and Date format is mismatch across blocks hence kept it here.